api/capi/include/nnstreamer-single.h

```diff
@@ -138,6 +138,18 @@ int ml_single_get_input_info (ml_single_h single, ml_tensors_info_h *info);
  */
 int ml_single_get_output_info (ml_single_h single, ml_tensors_info_h *info);
 
+/**
+ * @brief Sets the information (tensor dimension, type, name and so on) of required input data for the given model.
+ * @details Note that a model may not support setting such information.
+ * @param[in] single The model handle.
+ * @param[in] info The handle of input tensors information.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
```

Comment:
Let's add more description on this error, ```ML_ERROR_NOT_SUPPORTED```, for application developers:
```
Not supported. This implies that the given model or framework does not support dynamic dimensions. Use get input/output ..... APIs instead for this model and framework.
```

Reply:
Updated to :
```This implies that the given **~model or~** framework does not support dynamic dimensions. Use get input/output (( fill these in ..... )) APIs instead for **~this model and~** framework.```

There is no way to distinguish if the given input information set by the user is invalid or the model does allow setting dynamic dimensions. So, I think both these cases should be ML_ERROR_INVALID_PARAMETER. However, for framework, we can return ML_ERROR_NOT_SUPPORTED.